angular-workspace/components/scrollable/scrollable.decorator.ts

```diff
@@ -9,7 +9,7 @@ export const decorator: Addon_DecoratorFunction<StoryFnAngularReturnType> = (sto
     template: `
     <div
       id="scrollable-mock"
-      style="background-color: #efefef; height: 750px; max-width: 500px"
+      style="background-color: #efefef; display: flex; height: 750px; max-width: 500px"
```

Comment:
Waar is deze `display` voor? Hebben we een voorwaarde gecreeerd dat `<dso-scrollable>` alleen maar in `display: flex` kan staan?